fluss-flink/fluss-flink-common/src/test/java/com/alibaba/fluss/flink/sink/FlinkRowDataChannelComputerTest.java

```diff
@@ -28,15 +36,41 @@
 /** Test for {@link FlinkRowDataChannelComputer}. */
 class FlinkRowDataChannelComputerTest {
 
+    private static final FlussSerializationSchema<RowData> serializationSchema =
+            new RowSerializationSchema(false, false);
+
+    @BeforeAll
+    static void init() throws Exception {
+        serializationSchema.open(
+                new FlussSerializationSchema.InitializationContext() {
+                    @Override
+                    public MetricGroup getMetricGroup() {
+                        return null;
+                    }
+
+                    @Override
+                    public UserCodeClassLoader getUserCodeClassLoader() {
+                        return null;
```

Comment:
If we can't provide userClassloader and metricGroup in some cases, we shouldn't expose them to users in `FlussSerializationSchema.InitializationContext`, otherwise, user can't use it because it will break in the case of null. 

Currently, it's fine not expose them to users as they are rarely used.